book/embeds.md

```diff
@@ -496,7 +517,7 @@ Videos are similar to images, but demand more bandwidth, time, and
 memory; they also have complications like [Digital Rights Management
 (DRM)][drm]. The `<video>` tag addresses some of that, with built-in
 support for advanced video [*codecs*][codec]^[In video, it's called a
-"codec", but in images it's called a "format"--go figure.], DRM, and
+"codec", but in images it's called a "format",--go figure.] DRM, and
```

Comment:
This seems wrong. You meant to move it before the footnote?

Reply:
Oops, fixed in new PR